docs/tutorials/slo-validation-istio/basicusage.md

```diff
@@ -0,0 +1,135 @@
+---
+template: main.html
+---
+
+# Benchmark and Validate HTTP services
+
+The `slo-validation-istio` collects latency and error-related metrics and validates service-level objectives (SLOs) from an application using Istio.
+
+<!-- <p align='center'>
+  <img alt-text="load-test-http" src="../images/http-overview.png" width="90%" />
+</p> -->
+
+<!-- ***
+
+--8<-- "docs/tutorials/load-test-http/usecases.md"
+
+*** -->
+
+???+ warning "Before you begin"
+    <!-- Run the [httpbin](https://httpbin.org) sample service from a separate terminal.
+    ```shell
+    docker run -p 80:80 kennethreitz/httpbin
+    ```
+    You can also use [Podman](https://podman.io) or other alternatives to Docker in the above command. -->
+
+    Create a new Minikube cluster
+    ```shell
+    minikube start --memory=16384 --cpus=4 -p slo-validation-istio
+    ```
+
+    Install Istio
+    ```shell
+    istioctl install
+    ```
+
+    Install Prometheus plugin
+    ```shell
+    kubectl apply -f https://raw.githubusercontent.com/istio/istio/release-1.13/samples/addons/prometheus.yaml
+    ```
+
+    Enable automatic Istio sidecar injection
+    ```shell
+    kubectl label namespace default istio-injection=enabled --overwrite
+    ```
+
+    Run the httpbin sample service
+    ```shell
+    kubectl apply -f https://raw.githubusercontent.com/Alan-Cha/docs/slo-validation-istio/docs/tutorials/slo-validation-istio/samples/httpbin-v1-deployment.yaml
+    kubectl apply -f https://raw.githubusercontent.com/Alan-Cha/docs/slo-validation-istio/docs/tutorials/slo-validation-istio/samples/httpbin-service.yaml
+    ```
+
+    In a separate terminal window, run a Minikube tunnel 
+    ```shell
+    minikube tunnel -p slo-validation-istio
+    ```
+
+    In another separate terminal window, open the Prometheus API and allow local access to the Prometheus database
+    ```shell
+    istioctl dashboard prometheus
+    ```
+
+    ***
+
+    To send traffic, first run the sleep service
+    ```shell
+    kubectl apply -f https://raw.githubusercontent.com/Alan-Cha/docs/slo-validation-istio/docs/tutorials/slo-validation-istio/samples/sleep-deployment.yaml
+    ```
+
+    Then, make some requests
+    ```shell
+    export SLEEP_POD=$(kubectl get pod -l app=sleep -o jsonpath={.items..metadata.name})
+    kubectl exec "${SLEEP_POD}" -c sleep -- curl -sS http://httpbin:8000/headers
+    ```
+***
+
+## Collect metrics
+To collect metrics from a service, following must be set:
+
+- `endpoint`, the Prometheus database endpoint
+- `destination_workload`, the name of the service
+- `destination_workload_namespace`, the namespace of the service 
+- `startingTime`, the starting time from which to collect metrics (note the date format)
+
+```shell
+iter8 launch -c slo-validation-istio \
+--set endpoint=http://localhost:9090 \
+--set destination_workload=httpbin-v1 \
+--set destination_workload_namespace=default \
+--set startingTime="Feb 4\, 2014 at 6:05pm (PST)"
+```
```

Comment:
Because we have not included `slo-validation-istio` in `iter8lib/` yet, this will not work. Try the following:

```shell
git clone https://github.com/Alan-Cha/iter8.git iter8-alan
cd iter8-alan
git checkout istio-SLO-experiment-gen3
iter8 launch -c slo-validation-istio \
--noDownload \
--set endpoint=http://localhost:9090 \
--set destination_workload=httpbin-v1 \
--set destination_workload_namespace=default \
--set startingTime="Feb 4\, 2014 at 6:05pm (PST)"
```

Reply:
I found I needed to use  `go run main.go launch ...` instead of `iter8 launch ...` after checking out your repo.